links/src/main/java/juja/microservices/links/repository/impl/LinksRepositoryImpl.java

```diff
@@ -51,6 +51,10 @@ public Link getLinkByURL(String url) throws NotFoundException {
 
     @Override
     public List<Link> getAllLinks() {
```

Comment:
Maybe is better naming getAllAcriveLinks?

and name will be  without side effect 

what do you think?

Reply:
You mean Active or Archive?)) 
getAllActiveLinks good

Reply:
yes, you're right)  getAllActiveLinks or findAllActiveLinks